src/deserializer.rs

```diff
@@ -312,14 +312,19 @@ impl<'r> DeRecord<'r> for DeByteRecord<'r> {
 }
 
 macro_rules! deserialize_int {
-    ($method:ident, $visit:ident) => {
+    ($method:ident, $visit:ident, $inttype:ty) => {
         fn $method<V: Visitor<'de>>(
             self,
             visitor: V,
         ) -> Result<V::Value, Self::Error> {
-            visitor.$visit(
-                self.next_field()?
-                    .parse().map_err(|err| self.error(DEK::ParseInt(err)))?)
+            let field = self.next_field()?; 
+            let num = 
+                if field.starts_with("0x") {
+                    <$inttype>::from_str_radix( &field[2..], 16 )
+                } else {
+                    field.parse()
+                };
+            visitor.$visit( num.map_err(|err| self.error(DEK::ParseInt(err)))?)
```

Comment:
Sorry I missed this before, but please remove the spaces after an opening paren and before a closing one (as above). In general, it's a good idea to follow a style that's consistent with the surrounding code (which does not have these spaces).

Thanks!

Reply:
My apologies.  I haven't developed Rust style sensitivites yet :).